README.rst

```diff
@@ -190,6 +190,7 @@ Usage
      # issue_regex = REGEX (replace default `[A-Z]+-[0-9]+` regular expression)
      # resolved_statuses = comma separated list of statuses (closed, resolved)
      # run_test_case = True (default value for 'run' parameter)
+     # error_strategy [strict[skip|ignore] Choose how to handle connection errors
```

Comment:
1) typo, I believe there should be `[strict|skip|ignore]`

2) don't you mind to change it to `connection_error_strategy` ? or something similar, I think it is good to express that error is related to connection.

Reply:
Sure, do you want me to change only the outer param wording on the internal variable and attribute names as well?

Reply:
> Sure, do you want me to change only the outer param wording on the internal variable and attribute names as well?

In my opinion it would be better to change it on other places too, but it is up to you.

Reply:
You still have `[` instead of `|` after `strict` word in `[strict[skip|ignore]`.